### ISSUE TITLE
Missing Test Case - 443. String Compression

### ISSUE THREAD

**Issue by yanrs17** [LeetCode-Feedback/LeetCode-Feedback#4576]
#### Your LeetCode username

yanrs17

#### Category of the bug
- [ ] Missing Test Cases 

#### Code you used for Submit/Run operation

```
class Solution {
    public int compress(char[] chars) {

        char current = chars[0];
        int count = 0;
        int result = 0;
        int i = 0;

        for (char c : chars) {
            if (c == current) {
                count++;
            } else {
                chars[i++] = current;
                if (count > 1) {
                    for (char each : String.valueOf(count).toCharArray()) {
                        chars[i++] = each;
                    }
                }
                count = 1;
                current = c;
            }
            if (count == 1 || count == 2 || count == 10) {
                result++;
            }
        }
        chars[i++] = current;
        if (count > 1) {
            for (char each : String.valueOf(count).toCharArray()) {
                chars[i++] = each;
            }
        }

        return result;
    }
}
```

#### Language used for code

Java

#### Expected behavior

Actual: Code get accepted
Expected: Reject with error with this test case: `["a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a","a"]`
Basically 100 'a's. This test case is valid since it does not violate the constraints.

#### Screenshots
![image](https://user-images.githubusercontent.com/18586504/130337894-0764b298-3066-4113-b0bc-af69f759cc2d.png)

#### Additional context

Similarly, ["a"] * 1000 (repeat 1000 times) will also be false positive.
But I think there is also an alternative way: Simply add a constraint in the problem: "There will be at most 99 consecutive characters with the same value" etc. And pre-check when running the code.

**Comment by lc-kay** [comment]
Hi @yanrs17 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.
